docs/content/release-notes/flink-1.15.md

```diff
@@ -406,6 +406,18 @@ updating the client dependency to a version >= 7.14.0 is required due to interna
 The old JDBC connector (indicated by `connector.type=jdbc` in DDL) has been removed.
 If not done already, users need to upgrade to the newer stack (indicated by `connector=jdbc` in DDL).
 
+
+#### Extensible unified Sink uses new metric to capture outgoing records
+
+##### [FLINK-26126](https://issues.apache.org/jira/browse/FLINK-26126)
+
+New metrics `numRecordsSend` and `numRecordsSendErrors` has been introduced for users to monitor the number of 
+records sent to the external system. The `numRecordsOut` should be used to monitor the number of records 
+transferred between sink tasks.
+
+Connector developers should Consider using the new metric `numRecordsSendErrors` to monitor errors, 
```

Comment:
```suggestion
Connector developers should consider using the new metric `numRecordsSendErrors` to register errors when sending to the external system instead of `numRecordsOutErrors` which is deprecated from now on.
```